### DISCUSSION TITLE
Only start polling/listening on event like click

### DISCUSSION THREAD

**Discussion by I-OutOfGravity-I** [elkowar/eww#977] [General]
Is it possible to only start polling and listening to a script when for example a revealer reveals its content, to save computing resources?

**Comment by w-lfchen** [comment]
If you can create a boolean condition, that should be possible.
Simply use `:run-while` within your `defpoll` declaration.
See also [the eww documentation on this](https://elkowar.github.io/eww/configuration.html#adding-dynamic-content)

**Comment by I-OutOfGravity-I** [comment]
Jesus i am blind, thank you :)
